invenio_vocabularies/webpack.py

```diff
@@ -20,7 +20,7 @@
         "semantic-ui": dict(
             entry={},
             dependencies={
-                "@babel/runtime": "^7.9.0",
+                "@babel/runtime": "7.26.10",
```

Comment:
Do we really want to pin this version? I haven't seen it in any of the other rdm repos.